Mutagen.Bethesda.Analyzers.Skyrim/Record/Cell/Interior/EnemyLevelMultiplierAnalyzer.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Mutagen.Bethesda.Analyzers.SDK.Analyzers;
+using Mutagen.Bethesda.Analyzers.SDK.Topics;
+using Mutagen.Bethesda.Skyrim;
+
+namespace Mutagen.Bethesda.Analyzers.Skyrim.Record.Cell.Interior;
+
+public class EnemyLevelMultiplierAnalyzer : IContextualRecordAnalyzer<ICellGetter>
```

Comment:
This is the sort of analyzer im a bit wary of.  Too subjective.  Why 30%?  How would they change this if they wanted 40%?  What if someone is designing something to be a specific situation?  

Probably fine to keep.  But I do think we'll want to figure out some way to extract out analyzers to other packages so we can keep the "core" ones mostly objective

Reply:
This is basically a reminder in case you don't have any npcs in your dungeon that are not at the exact same level as the player, but also some that are maybe a bit higher level and more challenging.

The 30% are totally arbitrary and I agree it would be nice if you'd have options for something like this in the future to set it to 40% or whatever you want. Could be some kind of extension of `TopicConfig`.